test/integration/serverless/snapshots/log-csharp

```diff
@@ -1,170 +1,182 @@
 [
   {
     "message": {
-      "message": "START RequestId: XXX Version: $LATEST",
+      "message": "END RequestId: XXX",
       "lambda": {
-        "arn": "arn:aws:lambda:eu-west-1:425362996713:function:integration-tests-extension-STAGE-log-csharp",
+        "arn": "arn:aws:lambda:eu-west-1:425362996713:function:integration-tests-extension-XXXXXX-log-csharp",
         "request_id": "XXX"
       }
     },
     "status": "info",
-    "hostname": "arn:aws:lambda:eu-west-1:425362996713:function:integration-tests-extension-STAGE-log-csharp",
+    "timestamp": "XXX",
```

Comment:
The `timestamp` key was added to all the log type tests. Originally, the perl regex's were removing it entirely. Instead, I've opted for just normalizing the timestamp value instead of removing the whole thing.